mount_test.go

```diff
@@ -140,6 +140,75 @@ func Test_App_Mount_Express_Behavior(t *testing.T) {
 	utils.AssertEqual(t, uint32(16+9), app.routesCount)
 }
 
+// go test -run Test_App_Mount_RoutePositions
+func Test_App_Mount_RoutePositions(t *testing.T) {
+	t.Parallel()
+	testEndpoint := func(app *App, route, expectedBody string) {
+		resp, err := app.Test(httptest.NewRequest(MethodGet, route, nil))
```

Comment:
```suggestion
		resp, err := app.Test(httptest.NewRequest(MethodGet, route, http.NoBody))
```

This should have been caught by golangci-lint.